Library/.rubocop.yml

```diff
@@ -12,7 +12,7 @@ inherit_mode:
     - Exclude
 
 AllCops:
-  TargetRubyVersion: 2.6
+  TargetRubyVersion: 3.1
```

Comment:
This will need to be in a new PR as it enables a bunch of other changes that we don't want to fix yet.

Reply:
I dropped this commit.